_includes/header.html

```diff
@@ -11,7 +11,11 @@
       <a class="nav-link" href="sign_up_form.html">Sign up form</a>
     </li>
     <li class="nav-item">
+<<<<<<< HEAD
```

Comment:
@eddiejaoude having a bit of a nightmare with you & feeling confused about what's happened here. Can I save this for our call tomorrow? I worry I'm going to make this worse!